src/docfx/build/xref/XRefSpec.cs

```diff
@@ -20,10 +21,24 @@ public class XrefSpec
         [JsonExtensionData]
         public JObject ExtensionData { get; } = new JObject();
```

Comment:
This class feels like a mix of external xrefspec and internal xrefspec, may be we could separate the two.

Reply:
Sure, we can think about splitting them, but maybe in another PR